rust/main/chains/hyperlane-ethereum/src/contracts/multicall.rs

```diff
@@ -85,15 +119,13 @@ where
                 None
             }
         };
-        multicall.add_call(call, ALLOW_BATCH_FAILURES);
     });
```

Comment:
This aggregation is not relevant anymore since we removed population of individual transaction parameters in this PR - https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/5968/files#diff-5f49b3c719e488330745b8f07595760a4c875fc78e39621fdefb97bc0c8e6b63L617-L630

Reply:
We brought the aggregation back in this PR.